.github/workflows/workflow.yml

```diff
@@ -97,7 +97,7 @@ jobs:
           path: cypress/screenshots
 
       - name: build
-        run: yarn build
+        run: NODE_OPTIONS="--max-old-space-size=2048" yarn build
```

Comment:
@constanzauanini I added this for now to see if it helps (see [stackoverflow](https://stackoverflow.com/questions/53230823/fatal-error-ineffective-mark-compacts-near-heap-limit-allocation-failed-javas)) we could still debug the issue later

Reply:
I almost at the same time created a PR to increase memory

Reply:
https://github.com/terraware/terraware-web/pull/1721

Reply:
Saw that. Should we just do this for github workflows (where the problem exists) versus for everyone that runs the build?